RazorGenerator.Mvc/PrecompiledMvcEngine.cs

```diff
@@ -33,53 +33,46 @@ public PrecompiledMvcEngine(Assembly assembly, string baseVirtualPath)
         {
         }
 
-        public PrecompiledMvcEngine(Assembly assembly, string baseVirtualPath, IViewPageActivator viewPageActivator)
+        public static string[] DefaultAreaLocationFormats()
         {
-            AssemblyLastWriteTime = new Lazy<DateTime>(() => assembly.GetLastWriteTimeUtc(fallback: DateTime.MaxValue));
-            BaseVirtualPath = NormalizeBaseVirtualPath(baseVirtualPath);
-
-            base.AreaViewLocationFormats = new[] {
-                "~/Areas/{2}/Views/{1}/{0}.cshtml", 
-                "~/Areas/{2}/Views/{1}/{0}.vbhtml", 
-                "~/Areas/{2}/Views/Shared/{0}.cshtml", 
-                "~/Areas/{2}/Views/Shared/{0}.vbhtml", 
+            return new[] {
+                "~/Areas/{2}/Views/{1}/{0}.cshtml",
+                "~/Areas/{2}/Views/{1}/{0}.vbhtml",
+                "~/Areas/{2}/Views/Shared/{0}.cshtml",
+                "~/Areas/{2}/Views/Shared/{0}.vbhtml",
             };
+        }
 
-            base.AreaMasterLocationFormats = new[] {
-                "~/Areas/{2}/Views/{1}/{0}.cshtml", 
-                "~/Areas/{2}/Views/{1}/{0}.vbhtml", 
-                "~/Areas/{2}/Views/Shared/{0}.cshtml", 
-                "~/Areas/{2}/Views/Shared/{0}.vbhtml", 
+        public static string[] DefaultLocationFormats()
```

Comment:
Sorry - I don't use Git very often and don't think that committed the way I wanted it to, but that is the changes as requested to make these readonly fields.